src/libraries/System.Net.Sockets/ref/System.Net.Sockets.cs

```diff
@@ -726,6 +726,7 @@ public partial class TcpListener
         public void Start() { }
         public void Start(int backlog) { }
         public void Stop() { }
+        public void Dispose() { }
```

Comment:
Please put in alpha order (as when anyone updates with the special command it will do this)